### DISCUSSION TITLE
What is the default memory and cpu resource allocated to a pod by kubernetes?

### DISCUSSION THREAD

**Discussion by Zruto** [nextflow-io/nextflow#4029] [Q&A]
When I use nextflow and do not specify cpus and memory, I found that `request:cpu=2, memory=12Gi` in the created pod. Is this value related to the nextflow version or the kubernetes version? My nextflow version is 22.11.1 and kubernetes version is 1.23.0

**Comment by bentsherman** [comment]
By default, Nextflow doesn't specify any resources for CPUs and memory. I would double check that the resource settings aren't being insert from any config files. Beyond that, your K8s cluster might be imposing default resource requests, although a default of 2 cpus + 12gb memory is unusually high.
